jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/DefaultGeneratorManagerTest.java

```diff
@@ -43,6 +44,7 @@ void setUp() {
     GeneratorContext generatorContext = GeneratorContext.builder()
         .config(processorConfig)
         .logger(logger)
+        .project(new JavaProject(null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null))
```

Comment:
Why is this change needed? How is this change verifying the bug this patch is fixing?

Reply:
Thank you for your very valuable feedback! I take into account your remarks.